l10n_br_hr/__openerp__.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# (c) 2014 Kmee - Luis Felipe Mileo <mileo@kmee.com.br>
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+{
+    'name': 'Brazilian Localization HR',
+    'description': """
+Brazilian Localization HR with informations refered
+to the national context of HR""",
+    'category': 'Localization',
+    'author': 'KMEE',
```

Comment:
You can add here "Odoo Community Association (OCA)" like this https://github.com/OCA/l10n-brazil/blob/8.0/l10n_br_account_product/__openerp__.py#L10 this helps in Community statistics https://www.odoo.com/apps/modules/statistics